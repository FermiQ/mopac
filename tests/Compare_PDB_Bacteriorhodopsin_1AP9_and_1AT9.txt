#
# Script for use with the HTML file "Compare_PDB_Bacteriorhodopsin_1AP9_and_1AT9.html"
#
