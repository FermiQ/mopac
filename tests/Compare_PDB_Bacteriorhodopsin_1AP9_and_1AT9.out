 *******************************************************************************
 ** Site#:    0         For non-commercial use only    Version 19.136L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.136L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Thu May 23 17:08:03 2019  *
 *  COMPARE    - Compare two geometries
 *  GEO_DAT    - DATA SET GEOMETRY IS IN FILE "ADD-H_BACTERIORHODOPSIN_1AP9.ARC"
 *  GEO_REF    - REFERENCE GEOMETRY IS IN FILE "ADD-H_BACTERIORHODOPSIN_1AT9.ARC"
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  OUTPUT     - MINIMIZE OUTPUT
 *******************************************************************************
geo_dat="ADD-H_Bacteriorhodopsin_1AP9.arc" geo_ref="ADD-H_Bacteriorhodopsin_1AT9.arc" Compare output
 Compare 1AP9 and 1AT9 Bacteriorhodopsin

 

                            Differences in atoms sets

          Atoms in GEO_DAT only                   Atoms in GEO_REF only

       ATOM      8 2H   ARG A   7              ATOM      1  N   ALA A   2
       ATOM     14 3HG  ARG A   7              ATOM      2  CA  ALA A   2
       HETATM 3292  C15 RET   300              ATOM      3  C   ALA A   2
       HETATM 3293  C14 RET   300              ATOM      4  O   ALA A   2
       HETATM 3294  C13 RET   300              ATOM      5  CB  ALA A   2
       HETATM 3295  C12 RET   300              ATOM      6 1H   ALA A   2
       HETATM 3296  C11 RET   300              ATOM      7 2H   ALA A   2
       HETATM 3297  C10 RET   300              ATOM      8  HA  ALA A   2
       HETATM 3298  C9  RET   300              ATOM      9 1HB  ALA A   2
       HETATM 3299  C8  RET   300              ATOM     10 2HB  ALA A   2
       HETATM 3300  C7  RET   300              ATOM     11 3HB  ALA A   2
       HETATM 3301  C6  RET   300              ATOM     12  N   GLN A   3
       HETATM 3302  C1  RET   300              ATOM     13  CA  GLN A   3
       HETATM 3303  C2  RET   300              ATOM     14  C   GLN A   3
       HETATM 3304  C3  RET   300              ATOM     15  O   GLN A   3
       HETATM 3305  C4  RET   300              ATOM     16  CB  GLN A   3
       HETATM 3306  C20 RET   300              ATOM     17  CG  GLN A   3
       HETATM 3307  C5  RET   300              ATOM     18  CD  GLN A   3
       HETATM 3308  C19 RET   300              ATOM     19  OE1 GLN A   3
       HETATM 3309  C16 RET   300              ATOM     20  NE2 GLN A   3
       HETATM 3310  C17 RET   300              ATOM     21  H   GLN A   3
       HETATM 3311  C18 RET   300              ATOM     22  HA  GLN A   3
       HETATM 3322  H15 RET   300              ATOM     23 1HB  GLN A   3
       HETATM 3323  H14 RET   300              ATOM     24 2HB  GLN A   3
       HETATM 3324  H12 RET   300              ATOM     25 1HG  GLN A   3
       HETATM 3325  H11 RET   300              ATOM     26 2HG  GLN A   3
       HETATM 3326  H10 RET   300              ATOM     27 1HE2 GLN A   3
       HETATM 3327  H8  RET   300              ATOM     28 2HE2 GLN A   3
       HETATM 3328  H7  RET   300              ATOM     29  N   ILE A   4
       HETATM 3329 1H2  RET   300              ATOM     30  CA  ILE A   4
       HETATM 3330 2H2  RET   300              ATOM     31  C   ILE A   4
       HETATM 3331 1H3  RET   300              ATOM     32  O   ILE A   4
       HETATM 3332 2H3  RET   300              ATOM     33  CB  ILE A   4
       HETATM 3333 1H4  RET   300              ATOM     34  CG1 ILE A   4
       HETATM 3334 2H4  RET   300              ATOM     35  CG2 ILE A   4
       HETATM 3335 1H20 RET   300              ATOM     36  CD1 ILE A   4
       HETATM 3336 2H20 RET   300              ATOM     37  H   ILE A   4
       HETATM 3337 3H20 RET   300              ATOM     38  HA  ILE A   4
       HETATM 3338 1H19 RET   300              ATOM     39  HB  ILE A   4
       HETATM 3339 2H19 RET   300              ATOM     40 1HG1 ILE A   4
       HETATM 3340 3H19 RET   300              ATOM     41 2HG1 ILE A   4
       HETATM 3341 1H16 RET   300              ATOM     42 1HG2 ILE A   4
       HETATM 3342 2H16 RET   300              ATOM     43 2HG2 ILE A   4
       HETATM 3343 3H16 RET   300              ATOM     44 3HG2 ILE A   4
       HETATM 3344 1H17 RET   300              ATOM     45 1HD1 ILE A   4
       HETATM 3345 2H17 RET   300              ATOM     46 2HD1 ILE A   4
       HETATM 3346 3H17 RET   300              ATOM     47 3HD1 ILE A   4
       HETATM 3347 1H18 RET   300              ATOM     48  N   THR A   5
       HETATM 3348 2H18 RET   300              ATOM     49  CA  THR A   5
       HETATM 3349 3H18 RET   300              ATOM     50  C   THR A   5
       ATOM   3498 1HXT ARG A 225              ATOM     51  O   THR A   5
       ATOM   3499 2HXT ARG A 225              ATOM     52  CB  THR A   5
       ATOM   3500 3HXT ARG A 225              ATOM     53  OG1 THR A   5
       HETATM 3502  O   HOH   400              ATOM     54  CG2 THR A   5
       HETATM 3503 1H   HOH   400              ATOM     55  H   THR A   5
       HETATM 3504 2H   HOH   400              ATOM     56  HA  THR A   5
       HETATM 3505  O   HOH   401              ATOM     57  HB  THR A   5
       HETATM 3506 1H   HOH   401              ATOM     58  HG1 THR A   5
       HETATM 3507 2H   HOH   401              ATOM     59 1HG2 THR A   5
       HETATM 3508  O   HOH   402              ATOM     60 2HG2 THR A   5
       HETATM 3509 1H   HOH   402              ATOM     61 3HG2 THR A   5
       HETATM 3510 2H   HOH   402              ATOM     62  N   GLY A   6
       HETATM 3511  O   HOH   403              ATOM     63  CA  GLY A   6
       HETATM 3512 1H   HOH   403              ATOM     64  C   GLY A   6
       HETATM 3513 2H   HOH   403              ATOM     65  O   GLY A   6
       HETATM 3514  O   HOH   404              ATOM     66  H   GLY A   6
       HETATM 3515 1H   HOH   404              ATOM     67 1HA  GLY A   6
       HETATM 3516 2H   HOH   404              ATOM     68 2HA  GLY A   6
       HETATM 3517  O   HOH   405              ATOM     75  CD  ARG A   7
       HETATM 3518 1H   HOH   405              ATOM     76  NE  ARG A   7
       HETATM 3519 2H   HOH   405              ATOM     77  CZ  ARG A   7
       HETATM 3520  O   HOH   406              ATOM     78  NH1 ARG A   7
       HETATM 3521 1H   HOH   406              ATOM     79  NH2 ARG A   7
       HETATM 3522 2H   HOH   406              ATOM     86 1HD  ARG A   7
       HETATM 3523  O   HOH   407              ATOM     87 2HD  ARG A   7
       HETATM 3524 1H   HOH   407              ATOM     88  HE  ARG A   7
       HETATM 3525 2H   HOH   407              ATOM     89 1HH1 ARG A   7
       HETATM 3526  O   HOH   408              ATOM     90 2HH1 ARG A   7
       HETATM 3527 1H   HOH   408              ATOM     91  HH2 ARG A   7
       HETATM 3528 2H   HOH   408              HETATM 3369  C15 RET   216A
       HETATM 3529  O   HOH   409              HETATM 3370  C14 RET   216A
       HETATM 3530 1H   HOH   409              HETATM 3371  C13 RET   216A
       HETATM 3531 2H   HOH   409              HETATM 3372  C12 RET   216A
       HETATM 3532  O   HOH   410              HETATM 3373  C11 RET   216A
       HETATM 3533 1H   HOH   410              HETATM 3374  C10 RET   216A
       HETATM 3534 2H   HOH   410              HETATM 3375  C9  RET   216A
       HETATM 3535  O   HOH   411              HETATM 3376  C8  RET   216A
       HETATM 3536 1H   HOH   411              HETATM 3377  C7  RET   216A
       HETATM 3537 2H   HOH   411              HETATM 3378  C6  RET   216A
       HETATM 3538  O   HOH   412              HETATM 3379  C1  RET   216A
       HETATM 3539 1H   HOH   412              HETATM 3380  C2  RET   216A
       HETATM 3540 2H   HOH   412              HETATM 3381  C3  RET   216A
       HETATM 3541  O   HOH   413              HETATM 3382  C4  RET   216A
       HETATM 3542 1H   HOH   413              HETATM 3383  C20 RET   216A
       HETATM 3543 2H   HOH   413              HETATM 3384  C5  RET   216A
       HETATM 3544  O   HOH   414              HETATM 3385  C19 RET   216A
       HETATM 3545 1H   HOH   414              HETATM 3386  C16 RET   216A
       HETATM 3546 2H   HOH   414              HETATM 3387  C17 RET   216A
       HETATM 3547  O   HOH   415              HETATM 3388  C18 RET   216A
       HETATM 3548 1H   HOH   415              ATOM   3399  HZ  LYS A 216
       HETATM 3549 2H   HOH   415              HETATM 3400  H15 RET   216A
       HETATM 3550  O   HOH   416              HETATM 3401  H14 RET   216A
       HETATM 3551 1H   HOH   416              HETATM 3402  H12 RET   216A
       HETATM 3552 2H   HOH   416              HETATM 3403  H11 RET   216A
       HETATM 3553  O   HOH   417              HETATM 3404  H10 RET   216A
       HETATM 3554 1H   HOH   417              HETATM 3405  H8  RET   216A
       HETATM 3555 2H   HOH   417              HETATM 3406  H7  RET   216A
       HETATM 3556  O   HOH   418              HETATM 3407 1H2  RET   216A
       HETATM 3557 1H   HOH   418              HETATM 3408 2H2  RET   216A
       HETATM 3558 2H   HOH   418              HETATM 3409 1H3  RET   216A
       HETATM 3559  O   HOH   419              HETATM 3410 2H3  RET   216A
       HETATM 3560 1H   HOH   419              HETATM 3411 1H4  RET   216A
       HETATM 3561 2H   HOH   419              HETATM 3412 2H4  RET   216A
       HETATM 3562  O   HOH   420              HETATM 3413 1H20 RET   216A
       HETATM 3563 1H   HOH   420              HETATM 3414 2H20 RET   216A
       HETATM 3564 2H   HOH   420              HETATM 3415 3H20 RET   216A
       HETATM 3565  O   HOH   421              HETATM 3416 1H19 RET   216A
       HETATM 3566 1H   HOH   421              HETATM 3417 2H19 RET   216A
       HETATM 3567 2H   HOH   421              HETATM 3418 3H19 RET   216A
       HETATM 3568  O   HOH   422              HETATM 3419 1H16 RET   216A
       HETATM 3569 1H   HOH   422              HETATM 3420 2H16 RET   216A
       HETATM 3570 2H   HOH   422              HETATM 3421 3H16 RET   216A
       HETATM 3571  O   HOH   423              HETATM 3422 1H17 RET   216A
       HETATM 3572 1H   HOH   423              HETATM 3423 2H17 RET   216A
       HETATM 3573 2H   HOH   423              HETATM 3424 3H17 RET   216A
       HETATM 3574  O   HOH   424              HETATM 3425 1H18 RET   216A
       HETATM 3575 1H   HOH   424              HETATM 3426 2H18 RET   216A
       HETATM 3576 2H   HOH   424              HETATM 3427 3H18 RET   216A
       HETATM 3577  O   HOH   425              ATOM   3557  O   ARG A 225
       HETATM 3578 1H   HOH   425              ATOM   3577  N   SER A 226
       HETATM 3579 2H   HOH   425              ATOM   3578  CA  SER A 226
                                               ATOM   3579  C   SER A 226
                                               ATOM   3580  O   SER A 226
                                               ATOM   3581  CB  SER A 226
                                               ATOM   3582  OG  SER A 226
                                               ATOM   3583  H   SER A 226
                                               ATOM   3584  HA  SER A 226
                                               ATOM   3585 1HB  SER A 226
                                               ATOM   3586 2HB  SER A 226
                                               ATOM   3587  HG  SER A 226
                                               ATOM   3588  N   ARG A 227
                                               ATOM   3589  CA  ARG A 227
                                               ATOM   3590  C   ARG A 227
                                               ATOM   3591  O   ARG A 227
                                               ATOM   3592  CB  ARG A 227
                                               ATOM   3593  CG  ARG A 227
                                               ATOM   3594  CD  ARG A 227
                                               ATOM   3595  NE  ARG A 227
                                               ATOM   3596  CZ  ARG A 227
                                               ATOM   3597  NH1 ARG A 227
                                               ATOM   3598  NH2 ARG A 227
                                               ATOM   3599  H   ARG A 227
                                               ATOM   3600  HA  ARG A 227
                                               ATOM   3601 1HB  ARG A 227
                                               ATOM   3602 2HB  ARG A 227
                                               ATOM   3603 1HG  ARG A 227
                                               ATOM   3604 2HG  ARG A 227
                                               ATOM   3605 1HD  ARG A 227
                                               ATOM   3606 2HD  ARG A 227
                                               ATOM   3607  HE  ARG A 227
                                               ATOM   3608 1HH1 ARG A 227
                                               ATOM   3609 2HH1 ARG A 227
                                               ATOM   3610  HH2 ARG A 227
                                               ATOM   3611  N   ALA A 228
                                               ATOM   3612  CA  ALA A 228
                                               ATOM   3613  C   ALA A 228
                                               ATOM   3614  O   ALA A 228
                                               ATOM   3615  CB  ALA A 228
                                               ATOM   3616  H   ALA A 228
                                               ATOM   3617  HA  ALA A 228
                                               ATOM   3618 1HB  ALA A 228
                                               ATOM   3619 2HB  ALA A 228
                                               ATOM   3620 3HB  ALA A 228
                                               ATOM   3621  N   ILE A 229
                                               ATOM   3622  CA  ILE A 229
                                               ATOM   3623  C   ILE A 229
                                               ATOM   3624  O   ILE A 229
                                               ATOM   3625  CB  ILE A 229
                                               ATOM   3626  CG1 ILE A 229
                                               ATOM   3627  CG2 ILE A 229
                                               ATOM   3628  CD1 ILE A 229
                                               ATOM   3629  H   ILE A 229
                                               ATOM   3630  HA  ILE A 229
                                               ATOM   3631  HB  ILE A 229
                                               ATOM   3632 1HG1 ILE A 229
                                               ATOM   3633 2HG1 ILE A 229
                                               ATOM   3634 1HG2 ILE A 229
                                               ATOM   3635 2HG2 ILE A 229
                                               ATOM   3636 3HG2 ILE A 229
                                               ATOM   3637 1HD1 ILE A 229
                                               ATOM   3638 2HD1 ILE A 229
                                               ATOM   3639 3HD1 ILE A 229
                                               ATOM   3640  N   PHE A 230
                                               ATOM   3641  CA  PHE A 230
                                               ATOM   3642  C   PHE A 230
                                               ATOM   3643  O   PHE A 230
                                               ATOM   3644  CB  PHE A 230
                                               ATOM   3645  CG  PHE A 230
                                               ATOM   3646  CD1 PHE A 230
                                               ATOM   3647  CD2 PHE A 230
                                               ATOM   3648  CE1 PHE A 230
                                               ATOM   3649  CE2 PHE A 230
                                               ATOM   3650  CZ  PHE A 230
                                               ATOM   3651  H   PHE A 230
                                               ATOM   3652  HA  PHE A 230
                                               ATOM   3653 1HB  PHE A 230
                                               ATOM   3654 2HB  PHE A 230
                                               ATOM   3655  HD1 PHE A 230
                                               ATOM   3656  HD2 PHE A 230
                                               ATOM   3657  HE1 PHE A 230
                                               ATOM   3658  HE2 PHE A 230
                                               ATOM   3659  HZ  PHE A 230
                                               ATOM   3660  N   GLY A 231
                                               ATOM   3661  CA  GLY A 231
                                               ATOM   3662  C   GLY A 231
                                               ATOM   3663  O   GLY A 231
                                               ATOM   3664  H   GLY A 231
                                               ATOM   3665 1HA  GLY A 231
                                               ATOM   3666 2HA  GLY A 231
                                               ATOM   3667  HXT GLY A 231

                      Number of atoms in GEO_DAT: 3578
                      Number of atoms in GEO_REF: 3667
          Number of atoms common to both systems: 3447


                        After docking

                     Atoms that move a lot

                     Atom Label             GEO_REF Coordinates       Movement    Integral

            (ATOM   1919  HE2 TYR A 131)  17.394  19.307 -17.760       16.14        16.14
            (ATOM   1917  HD2 TYR A 131)  16.320  17.185 -17.019       14.68        30.81
            (ATOM   1909  CE2 TYR A 131)  16.330  19.141 -17.926       14.64        45.45
            (ATOM   1911  OH  TYR A 131)  16.146  21.299 -18.960       14.33        59.78
            (ATOM   1907  CD2 TYR A 131)  15.727  17.955 -17.513       13.82        73.60
            (ATOM   1920  HH  TYR A 131)  15.599  21.992 -19.407       13.56        87.16
            (ATOM   1910  CZ  TYR A 131)  15.564  20.116 -18.554       13.55       100.71
            (ATOM   1011  HZ  PHE A  71)  14.755   9.637 -16.208       13.05       113.76
            (ATOM   1024 1HA  GLY A  73)  14.446  14.872 -25.793       12.74       126.50
            (ATOM   1883 2HZ  LYS A 129)   3.668  10.465 -17.082       12.30       138.80
            (ATOM   1915 2HB  TYR A 131)  14.362  15.947 -16.526       12.21       151.01
            (ATOM   3433  HH2 ARG A 225)  30.206 -10.586  13.556       12.09       163.10
            (ATOM   1882 1HZ  LYS A 129)   2.899  11.222 -18.329       12.07       175.17
            (ATOM   3424  NH2 ARG A 225)  29.374 -10.210  14.010       11.96       187.13
            (ATOM   1905  CG  TYR A 131)  14.364  17.723 -17.718       11.96       199.09
            (ATOM   1057 1HE2 GLN A  75)   9.805   4.095 -24.075       11.90       210.99
            (ATOM   1908  CE1 TYR A 131)  14.206  19.907 -18.771       11.70       222.69
            (ATOM   1018 2HA  GLY A  72)  13.625  16.911 -21.739       11.69       234.38
            (ATOM   1914 1HB  TYR A 131)  13.623  15.760 -18.135       11.43       245.81
            (ATOM   1015  O   GLY A  72)  14.008  16.615 -24.221       11.41       257.23
            (ATOM   3434 1HH1 ARG A 225)  31.497  -9.009  14.595       11.37       268.59
            (ATOM   1010  HE2 PHE A  71)  12.322   9.291 -16.535       11.32       279.91
            (ATOM   1058 2HE2 GLN A  75)  10.909   3.748 -22.900       11.29       291.20
            (ATOM   1871  NZ  LYS A 129)   3.796  10.927 -17.971       11.23       302.42
            (ATOM    409 3HE  MET A  32)  20.656 -19.389  16.385       11.20       313.63
            (ATOM   1904  CB  TYR A 131)  13.730  16.423 -17.276       11.09       324.72
            (ATOM   1002  CZ  PHE A  71)  14.173  10.362 -16.777       11.09       335.81
            (ATOM   1881 2HE  LYS A 129)   4.585   9.027 -18.434       11.07       346.88
            (ATOM   1906  CD1 TYR A 131)  13.615  18.713 -18.353       10.95       357.82
            (ATOM    407 1HE  MET A  32)  20.835 -21.036  15.732       10.95       368.77

   Difference between GEO_DAT and GEO_REF:6250.041 = total,  1.8132 = Average,  2.6720 = RMS, in Angstroms

                             Differences between bond-lengths for the two geometries

           Diff.                 Atom 1                       Atom 2                  Bond length      Bond length
                                                                                      in GEO_DAT      in GEO_REF
   1       0.091     C(ATOM   2378  CA  LYS A 159)    C(ATOM   2379  C   LYS A 159)      1.528            1.437
   2       0.087     N(ATOM   2398  N   ALA A 160)    C(ATOM   2379  C   LYS A 159)      1.342            1.256
   3       0.068     N(ATOM   2398  N   ALA A 160)    C(ATOM   2399  CA  ALA A 160)      1.459            1.391
   4       0.059     N(ATOM   1953  N   ARG A 134)    C(ATOM   1934  C   TYR A 133)      1.326            1.386
   5       0.055     C(ATOM    460  CB  PRO A  37)    C(ATOM    461  CG  PRO A  37)      1.503            1.558
   6       0.053     N(ATOM   2434  N   MET A 163)    C(ATOM   2435  CA  MET A 163)      1.468            1.414
   7       0.048     C(ATOM    983  CG  PRO A  70)    C(ATOM    984  CD  PRO A  70)      1.482            1.530
   8       0.047     N(ATOM   2423  N   SER A 162)    C(ATOM   2410  C   GLU A 161)      1.330            1.282
   9       0.045     C(ATOM    982  CB  PRO A  70)    C(ATOM    983  CG  PRO A  70)      1.483            1.528
  10       0.045     N(ATOM   2434  N   MET A 163)    C(ATOM   2425  C   SER A 162)      1.331            1.286
  11       0.044     C(ATOM   2424  CA  SER A 162)    C(ATOM   2425  C   SER A 162)      1.528            1.483
  12       0.044     C(ATOM   2435  CA  MET A 163)    C(ATOM   2436  C   MET A 163)      1.539            1.495
  13       0.044     O(ATOM   2477  O   PRO A 165)    C(ATOM   2476  C   PRO A 165)      1.227            1.271
  14       0.043     O(ATOM   2496  OE2 GLU A 166)    C(ATOM   2494  CD  GLU A 166)      1.246            1.290
  15       0.042     O(ATOM   2415  OE1 GLU A 161)    C(ATOM   2414  CD  GLU A 161)      1.251            1.293
  16       0.042     C(ATOM     88  CB  TRP A  12)    C(ATOM     89  CG  TRP A  12)      1.536            1.494
  17       0.042     O(ATOM   2416  OE2 GLU A 161)    C(ATOM   2414  CD  GLU A 161)      1.251            1.293
  18       0.042     S(ATOM   2140  SD  MET A 145)    C(ATOM   2141  CE  MET A 145)      1.814            1.773
  19       0.042     C(ATOM   1700  CG1 ILE A 117)    C(ATOM   1702  CD1 ILE A 117)      1.552            1.510
  20       0.040     O(ATOM   2495  OE1 GLU A 166)    C(ATOM   2494  CD  GLU A 166)      1.250            1.290
  21       0.040     C(ATOM   2435  CA  MET A 163)    C(ATOM   2438  CB  MET A 163)      1.543            1.503
  22       0.039     C(ATOM   1077  CB  PRO A  77)    C(ATOM   1078  CG  PRO A  77)      1.494            1.533
  23       0.039     N(ATOM      1  N   ARG A   7)    C(ATOM      2  CA  ARG A   7)      1.500            1.461
  24       0.039     N(ATOM   1849  N   THR A 128)    C(ATOM   1850  CA  THR A 128)      1.463            1.502
  25       0.038     C(ATOM   1231  CB  TRP A  86)    C(ATOM   1232  CG  TRP A  86)      1.527            1.489

    GEO_DAT:  'ADD-H_Bacteriorhodopsin_1AP9.arc'
    GEO_REF:  'ADD-H_Bacteriorhodopsin_1AT9.arc'


                    Analysis of Non-Covalent Interactions

          Total non-covalent energy of GEO_DAT system:  -3285.29 Kcal/mol
          Total non-covalent energy of GEO_REF system:  -3357.90 Kcal/mol
                                           Difference:     72.60 Kcal/mol

          Number of hydrogen bonds common to both systems:  187


                            Hydrogen bonds in GEO_DAT but not in GEO_REF

               Donor atom                     Hydrogen atom            H-bond length(A)     Energy    Sum

   1   "ATOM   2948  OE1 GLU A 194"      "ATOM   1204  HH  TYR A  83"       1.727           -4.325   -4.325
   2   "ATOM    130  O   ALA A  14"      "ATOM    173  HG1 THR A  17"       1.642           -3.917   -8.242
   3   "ATOM   3228  OD2 ASP A 212"      "ATOM    791  HH  TYR A  57"       1.776           -3.403  -11.645
   4   "ATOM    948  O   MET A  68"      "ATOM   1856  H   THR A 128"       1.531           -2.940  -14.585
   5   "ATOM   3089  OE2 GLU A 204"      "ATOM   2936  H   SER A 193"       1.842           -2.735  -17.319
   6   "ATOM   1442  O   ALA A  98"      "ATOM   1521  H   ALA A 103"       2.075           -2.250  -19.570
   7   "ATOM    393  N   MET A  32"      "ATOM    511 1HZ  LYS A  40"       2.296           -2.120  -21.689
   8   "ATOM   1771  N   GLY A 122"      "ATOM   2099  HG  SER A 141"       2.109           -1.779  -23.468
   9   "ATOM   2904  N   ILE A 191"      "ATOM   2968  H   ALA A 196"       2.958           -1.769  -25.237
  10   "ATOM    694  O   ILE A  52"      "ATOM    762  H   MET A  56"       2.188           -1.730  -26.968
  11   "ATOM   3274  O   LYS A 216"      "ATOM   3337  H   GLY A 220"       1.603           -1.711  -28.678
  12   "ATOM   1195  OH  TYR A  83"      "ATOM   2880  HE1 TRP A 189"       1.586           -1.661  -30.340
  13   "ATOM    588  O   ILE A  45"      "ATOM    625  H   THR A  47"       1.880           -1.577  -31.916
  14   "ATOM    771  N   TYR A  57"      "ATOM    817  H   SER A  59"       2.895           -1.546  -33.462


                            Hydrogen bonds in GEO_REF but not in GEO_DAT

               Donor atom                     Hydrogen atom            H-bond length(A)     Energy    Sum

   1   "ATOM   1800  O   VAL A 124"      "ATOM   1859  HG1 THR A 128"       1.506           -4.373   -4.373
   2   "ATOM   1529  O   ASP A 104"      "ATOM   1575 3HG2 THR A 107"       1.567           -2.454   -6.827
   3   "ATOM     87  O   TRP A  12"      "ATOM    160  H   GLY A  16"       2.012           -2.272   -9.099
   4   "ATOM   1154  O   ALA A  81"      "ATOM   1223  H   ASP A  85"       2.131           -2.222  -11.321
   5   "ATOM    578  O   ALA A  44"      "ATOM    630 2HG2 THR A  47"       1.683           -2.122  -13.443
   6   "ATOM   1734  O   ILE A 119"      "ATOM   1786  H   LEU A 123"       1.948           -2.088  -15.531
   7   "ATOM   2357  OG1 THR A 157"      "ATOM   2650  HE  ARG A 175"       1.859           -2.063  -17.594
   8   "ATOM    743  O   THR A  55"      "ATOM    817  H   SER A  59"       2.223           -2.060  -19.654
   9   "ATOM    516  O   LYS A  41"      "ATOM    593  H   ILE A  45"       2.191           -1.960  -21.614
  10   "ATOM    908  O   GLY A  65"      "ATOM   1156  H   ALA A  81"       2.464           -1.885  -23.499
  11   "ATOM    557  O   TYR A  43"      "ATOM    625  H   THR A  47"       1.988           -1.879  -25.377
  12   "ATOM   2544  O   THR A 170"      "ATOM   2620  H   LEU A 174"       2.183           -1.870  -27.248
  13   "ATOM   3343  O   LEU A 221"      "ATOM   3417  H   ARG A 225"       2.021           -1.857  -29.104
  14   "ATOM   1090  O   ILE A  78"      "ATOM    938  H   THR A  67"       2.285           -1.841  -30.945
  15   "ATOM   1442  O   ALA A  98"      "ATOM   1511  H   ASP A 102"       1.729           -1.807  -32.752
  16   "ATOM   2832  O   VAL A 187"      "ATOM   2912  H   ILE A 191"       2.277           -1.793  -34.545
  17   "ATOM   1545  OE1 GLN A 105"      "ATOM   1547  H   GLN A 105"       1.835           -1.778  -36.323
  18   "ATOM   1208  O   ALA A  84"      "ATOM   1281  H   PHE A  88"       2.353           -1.737  -38.060
  19   "ATOM    111  O   LEU A  13"      "ATOM    173  HG1 THR A  17"       1.838           -1.735  -39.795
  20   "ATOM   2717  O   VAL A 180"      "ATOM   2789  H   ALA A 184"       2.279           -1.733  -41.528
  21   "ATOM   1335  O   LEU A  92"      "ATOM   1416  H   ASP A  96"       2.062           -1.717  -43.244
  22   "ATOM    621  O   THR A  47"      "ATOM    686  H   ALA A  51"       2.200           -1.653  -44.898
  23   "ATOM   1887  O   VAL A 130"      "ATOM   1964  H   ARG A 134"       2.457           -1.639  -46.537
  24   "ATOM   2634  O   ARG A 175"      "ATOM   2705  H   VAL A 179"       2.118           -1.609  -48.146
  25   "ATOM   1924  O   SER A 132"      "ATOM   2003  H   VAL A 136"       2.365           -1.549  -49.695
  26   "ATOM   2523  O   ALA A 168"      "ATOM   2584  H   LYS A 172"       2.293           -1.538  -51.233
  27   "ATOM   1373  O   LEU A  94"      "ATOM   1444  H   ALA A  98"       2.014           -1.495  -52.728
  28   "ATOM    995  O   PHE A  71"      "ATOM   1023  H   GLY A  73"       1.734           -1.492  -54.221
  29   "ATOM    557  O   TYR A  43"      "ATOM    614  HG1 THR A  46"       2.045           -1.481  -55.702
  30   "ATOM   2213  O   LEU A 149"      "ATOM   2296  H   PHE A 153"       2.231           -1.464  -57.166
  31   "ATOM   3018  O   PRO A 200"      "ATOM   3090  H   GLU A 204"       2.060           -1.436  -58.602
  32   "ATOM   2454  O   ARG A 164"      "ATOM   2525  H   ALA A 168"       2.262           -1.436  -60.039
  33   "ATOM   1903  O   TYR A 131"      "ATOM   1987  H   PHE A 135"       2.191           -1.431  -61.470


    The following files will be written:
    'Compare_PDB_Bacteriorhodopsin_1AP9_and_1AT9.html'
    'ADD-H_Bacteriorhodopsin_1AP9.pdb'
    'ADD-H_Bacteriorhodopsin_1AT9.pdb'

          GEO_REF with 0SCF: Job complete

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * GEO_REF with 0SCF: Job complete                                            *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
